doc/rst/src/tests.rst

```diff
@@ -0,0 +1,66 @@
+
+**********
+Unit tests
+**********
+
+This project uses `XCommon CMake` to build the unit tests in a similar fashion to the examples.
+
+Unit tests target the `XK-EVK-XU316` board and x86 platforms.
+All unit tests are located in the */tests/* directory:
+
+* `/tests/ <https://github.com/xmos/lib_xcore_math/tree/develop/tests/>`_ - Unit test projects for ``lib_xcore_math``:
```

Comment:
I don't like the links to develop, as existing documents break if this is changes in the future. But that's just my opinion, and I can't suggest an alternative link, so this is fine as it is.

Reply:
Agreed. I actually removed all links to GitHub in the library description. I left these ones as they are a bit more "developery"